packages/core-toggle/readme.md

```diff
@@ -44,6 +43,106 @@ Content is only toggled when clicking `button`. Great for accordions and expand/
+
+```html
+<div>
+  <button class="my-toggle" aria-controls="content">Toggle VanillaJS</button>
+</div>
+<div id="content" hidden>Content</div>
+```
+
+### Autofocus
+
+If you have form elements inside a `@nrk/core-toggle`, you can optionally add a `autofocus` attribute to the most prominent form element. This helps the user navigate quickly when toggle is opened.
+
+
+
+
+## Events
+
+### toggle
+
+Before a `@nrk/core-toggle` changes open state, a [toggle event](https://www.w3schools.com/jsref/event_ontoggle.asp) is fired (both for VanillaJS and React/Preact components). The toggle event is cancelable, meaning you can use [`event.preventDefault()`](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) to cancel toggling. The event also [bubbles](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#Event_bubbling_and_capture), and can therefore be detected both from the button element itself, or any parent element (read [event delegation](https://stackoverflow.com/questions/1687296/what-is-dom-event-delegation)):
+
+
+```js
+document.addEventListener('toggle', (event) => {
+  event.target                              // The button element triggering toggle event
+  event.detail.relatedTarget                // The content element controlled by button
+  event.detail.isOpen                       // The current toggle state (before toggle event has run)
+  event.detail.willOpen                     // The wanted toggle state
+})
+```
+
+### toggle.select
+
+The `toggle.select` event is fired whenever an item is selected inside a toggle with the `popup` option enabled.
+Useful for setting the value of the toggle button with the selected value.
+
+
+```js
+document.addEventListener('toggle.select', (event) => {
+  event.target                              // The buttom element triggering the event
+  event.detail.relatedTarget                // The content element controlled by button
+  event.detail.currentTarget                // The item element selected
+  event.detail.value                        // The selected item's value
+})
+```
+
+
+## Styling
+
+All styling in documentation is example only. Both the `<button>` and content element receive attributes reflecting the current toggle state:
+
+```css
+.my-toggle {}                         /* Target button in any state */
+.my-toggle[aria-expanded="true"] {}   /* Target only open button */
+.my-toggle[aria-expanded="false"] {}  /* Target only closed button */
+
+.my-toggle-content {}                 /* Target content in any state */
+.my-toggle-content:not([hidden]) {}   /* Target only open content */
+.my-toggle-content[hidden] {}         /* Target only closed content */
+```
+
 ## Demo: Popup
```

Comment:
Why is demo popup moved after everything? Popup is one of the most used features, so was therefore put before everything earlier

Reply:
for consitency, we have one demo on the top, usage after that ... and rest of demos before FAQ or similar to the end. the important part is that usage is not all the way in the middle of the document.

Reply:
👍 would suggest popup-example in top then :)